src/pages/workspace/reimburse/WorkspaceReimbursePage.js

```diff
@@ -24,13 +27,16 @@ const WorkspaceReimbursePage = props => (
         route={props.route}
         guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_REIMBURSE}
     >
-        {(hasVBA, policyID) => (
-            <WorkspaceReimburseView policyID={policyID} hasVBA={hasVBA} />
+        {hasVBA => (
+            <WorkspaceReimburseView policy={props.policy} hasVBA={hasVBA} />
```

Comment:
hmm, since we're using `withPolicy` in this file only to pass it to `WorkspaceReimburseView `, why not use it directly in `WorkspaceReimburseView`? NAB though.